### PR TITLE
Added basic boolean helpers

### PR DESCRIPTION
**Pull request by kspearrin**
Added boolean helpers Equal, NotEqual, and Not.

### CODE REVIEW

File: src/Handlebars.Net.Helpers.Core/Enums/Category.cs [hunk 1]
```diff
@@ -15,5 +15,6 @@ public enum Category
     Random,
     JsonPath,
     DynamicLinq,
-    Humanizer
+    Humanizer,
+    Boolean
```
Comment (StefH) [inline src/Handlebars.Net.Helpers.Core/Enums/Category.cs:19]: Can you also update the description in de .csproj to include Boolean ?

File: src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using HandlebarsDotNet.Helpers.Attributes;
+using HandlebarsDotNet.Helpers.Enums;
+
+namespace HandlebarsDotNet.Helpers.Helpers;
+
+internal class BooleanHelpers : BaseHelpers, IHelpers
+{
+    public BooleanHelpers(IHandlebars context) : base(context)
```
Comment (StefH) [inline src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs:8]: Should we also add all boolean operators?

Like "Xor" , "Or", and so on?

**Review by StefH — CHANGES_REQUESTED** [review]

File: src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs (hunk 2 above)
Comment (kspearrin) [reply src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs:8]: Not a bad idea. I mostly added this as a need for `Not`, but we can certainly expand.

File: src/Handlebars.Net.Helpers.Core/Enums/Category.cs (hunk 1 above)
Comment (kspearrin) [reply src/Handlebars.Net.Helpers.Core/Enums/Category.cs:19]: Done

File: src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs (hunk 2 above)
Comment (kspearrin) [reply src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs:8]: I added `Or`, `LogicalOr`, `LogicalXor`, `And`, and `LogicalAnd`.

See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators

**Review by StefH — APPROVED** [review]
thanks

**Comment by StefH** [comment]
@kspearrin 
Are there any other changes you would like to do? Or can I create a new NuGet version?

**Comment by kspearrin** [comment]
This is all for now. Thanks!

**Pull request merged**